engine/src/exec-extension.cpp

```diff
@@ -199,6 +195,10 @@ void MCEngineExecLoadExtension(MCExecContext& ctxt, MCStringRef p_filename, MCSt
     
     MCValueRelease(t_stream);
     
+    for(MCLoadedExtension *t_ext = MCextensions; t_ext != nil; t_ext = t_ext -> next)
```

Comment:
This can never happen - a module cannot be loaded / created by MCScriptCreateMOduleFromStream if there is one in memory with the same name.